modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -440,12 +440,13 @@ export async function putListTypeEntry({
 	});
 }
 
-export async function updateRelationship({
+export async function putObjectRelationship({
```

Comment:
@igor-franca check sorting here too.

Reply:
Checked! It has already been done in the sort commit